src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java

```diff
@@ -13,7 +13,7 @@
 
     @Test
     void givenString_whenSortJava8_thenSorted() {
-        String sorted = "bdCa".chars()
+        String sorted = "bdca".chars()
```

Comment:
I'm running java 8 (151), and when running the tests this failed. Was the uppercase C intentional?

Reply:
Really weird, can you provide exact error that you got? All tests are run on each build, so this definitely should not sneak in

Reply:
I ran the test from IntelliJ, the outcome is:
```
org.opentest4j.AssertionFailedError: 
Expecting:
 <"Cabd">
to be equal to:
 <"abcd">
but was not.
Expected :abcd
Actual   :Cabd
```

- I don't see how the implementation would yield `abcd` as there's nothing down-casing the character in the stream

Reply:
Ah - I think the problem is the CI job isn't running the tests, the output of (just picked one at random) Travis:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Results :
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

https://travis-ci.org/pivovarit/throwing-function/jobs/507548530